RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 17.9
 -----
+* [***] New Block: Embed block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3727]
```

Comment:
Not sure if it's standard but I noticed that other entries related to changes in the editor include `Block editor: ` message at the beginning, what do you think about including it too?

Reply:
That's a good point, I've added that in https://github.com/wordpress-mobile/WordPress-Android/pull/15054/commits/a4c0328130180d967eebb394eb4c03fac2e1b575, to bring this set of release notes in line with previous releases. 🙇‍♀️